Gemfile

```diff
@@ -1,14 +1,16 @@
 source "https://rubygems.org"
 
-gem "quke"
-
-# We use rubocop in all our Ruby based projects to try and ensure consistency
-# in the code we write across all our projects.
-gem "rubocop", require: false
+gem "quke",
```

Comment:
Are we sure we want to bypass the gem version and go straight to master here? Or shall we release a new version?

Reply:
I've lifted this from @Cruikshanks 's earlier PR: https://github.com/DEFRA/waste-carriers-acceptance-tests/pull/166/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478L7 and this is what we have on WEX too, so probably a question for Alan.

What do you think is the risk here? Is it that a Quke change may inadvertently break the tests?

Reply:
@Cruikshanks ?

Reply:
Ordinarily, @cintamani would be right. You'd avoid this and just go with released gems.

What I will often do though, is some minor tweaks to Quke as part of a general theme or feature. I don't want to cut a release until that thing is done, but our projects can still benefit from those tweaks.

So we've got in the habit of pulling from the repo rather than rubygems.

But in truth, there is nothing stopping me ramping up the versions of Quke for each tweak. If you waggle your fingers at me hard enough I'll pull my finger out and get better, and you can then follow the better practice of what you had before.

Reply:
Thanks both. I'm happy to trust Alan's judgement on this. If it's a minor tweak that's unlikely to break something then I'm happy for it to just get updated without the formalities of an updated version, with the option to roll back the version if the tests do ever break.